includes/definitions/ipolis.yaml

```diff
@@ -0,0 +1,9 @@
+os: ipolis
+text: 'Hanwha Techwin'
+type: camera
+icon: hanwhatechwin
+over: 
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.36849
```

Comment:
One last thing from me. That's quite a low level OID to use. I can see the full one is `.1.3.6.1.4.1.36849.1.2.88` - is it worth doing `.1.3.6.1.4.1.36849.1.2.` or `1.3.6.1.4.1.36849.1.`?